From: Tejun Heo <htejun@gmail.com>
Subject: ata_piix: add TECRA M4 to broken suspend list
References: 398270

TOSHIBA also used "TECRA M4" in additon to "Tecra M4", add it.

Signed-off-by: Tejun Heo <htejun@gmail.com>
Signed-off-by: Tejun Heo <teheo@suse.de>
---
 drivers/ata/ata_piix.c |    7 +++++++
 1 file changed, 7 insertions(+)

Index: linux-2.6.25-SL110_BRANCH/drivers/ata/ata_piix.c
===================================================================
--- linux-2.6.25-SL110_BRANCH.orig/drivers/ata/ata_piix.c
+++ linux-2.6.25-SL110_BRANCH/drivers/ata/ata_piix.c
@@ -1223,6 +1223,13 @@ static int piix_broken_suspend(void)
 			},
 		},
 		{
+			.ident = "TECRA M4",
+			.matches = {
+				DMI_MATCH(DMI_SYS_VENDOR, "TOSHIBA"),
+				DMI_MATCH(DMI_PRODUCT_NAME, "TECRA M4"),
+			},
+		},
+		{
 			.ident = "TECRA M5",
 			.matches = {
 				DMI_MATCH(DMI_SYS_VENDOR, "TOSHIBA"),
